cylc/flow/parsec/fileparse.py

```diff
@@ -441,6 +448,14 @@ def hashbang_and_plugin_templating_clash(
             f"{templating}"
             f" which does not match {flines[0]} set in flow.cylc."
         )
+    elif (
+        hashbang is None
+        and templating == 'template variables'
+    ):
+        raise TemplateVarLanguageClash(
+            'Plugins provided template variables, but workflow definition '
+            'has no hashbang (e.g. #jinja2): Templating will fail.'
```

Comment:
```suggestion
            'has no hashbang (e.g. #!jinja2): Templating will fail.'
```
Also, possibly "shebang" is more widely used?